server/ciphersuites_1.4.go

```diff
@@ -1,33 +0,0 @@
-// Copyright 2015 Apcera Inc. All rights reserved.
```

Comment:
I'll update the README.  I was debating on including 1.8 - it is beta today, but scheduled for release on 1/31/17 -  wdyt?